js/stores/windowStore.js

```diff
@@ -641,7 +648,7 @@ const doAction = (action) => {
       break
     case windowConstants.WINDOW_TOGGLE_MENUBAR_VISIBLE:
       if (getSetting(settings.AUTO_HIDE_MENU)) {
-        doAction({actionType: windowConstants.WINDOW_SET_CONTEXT_MENU_DETAIL})
+        windowState = contextMenuState.setContextMenu(windowState)
```

Comment:
there is some weird reason why these call the action instead of just setting state. To be safe I think we shouldn't change it unless we have to